src/sst/core/model/json/jsonmodel.cc

```diff
@@ -0,0 +1,390 @@
+// -*- c++ -*-
+
+// Copyright 2009-2021 NTESS. Under the terms
+// of Contract DE-NA0003525 with NTESS, the U.S.
+// Government retains certain rights in this software.
+//
+// Copyright (c) 2009-2021, NTESS
+// All rights reserved.
+//
+// This file is part of the SST software package. For license
+// information, see the LICENSE file in the top level directory of the
+// distribution.
+
+#include "sst/core/model/json/jsonmodel.h"
+
+#ifdef SST_CONFIG_HAVE_MPI
```

Comment:
Is there are reason to include mpi in this file?  I don't see MPI used anywhere.

Reply:
This was copied over from the python model reader, so I can remove it.

Reply:
removed in commit cc7003ef